client/testclient/cmd/lookup.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/coniks-sys/coniks-go/client"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	p "github.com/coniks-sys/coniks-go/protocol"
+	"github.com/spf13/cobra"
+)
+
+var lookupCmd = &cobra.Command{
+	Use:   "lookup",
+	Short: "Lookup a name.",
+	Long:  `Lookup the key of some known contact.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		conf := loadConfigOrExit(cmd)
+		name := cmd.Flag("name").Value.String()
+		if len(name) == 0 {
+			cmd.Usage()
+			return
+		}
+		req, err := createLookupRequest(name)
+		if err != nil {
+			fmt.Println("Couldn'r create request!")
+			os.Exit(-1)
+		}
+		resp, err := testutil.NewTCPClient(req)
+		if err != nil {
+			fmt.Println("Error while retrieving repsonse: " + err.Error())
+			os.Exit(-1)
+		}
+		response, errCode := client.UnmarshalResponse(p.KeyLookupType,
```

Comment:
Makes sense! Do you think this should be done in this PR, too? (or in a separate one)